packages/legend-application-studio/src/components/LegendStudioApplication.tsx

```diff
@@ -188,6 +189,15 @@ export const LegendStudioApplicationRoot = observer(() => {
               ]}
               component={WorkspaceSetup}
             />
+            <Route
+              exact={true}
+              path={[
+                // root path will lead to project overview
+                '/',
```

Comment:
this is bad, we already have `/` mapped to the setup page, please be careful here!

Reply:
!! my bad i did not comprehend, yep makes sense 👍